modules/nf-core/muse/sump/tests/wes.config

```diff
@@ -0,0 +1,8 @@
+process {
+
+    withName: 'MUSE_SUMP' {
+        ext.args    = '-E' 
+        stageInMode = 'copy'
```

Comment:
I think this wants to be in the main.nf, not here, to ensure the module works on cloud.

Reply:
I have no experience with when to add the stageInMode to the main and when to add it to the .config :/ just saw it like that all the time. I can move it though.